api/swagger.yaml

```diff
@@ -2102,6 +2102,16 @@ definitions:
                     SpreadDescriptor:
                       description: "label descriptor, such as engine.labels.az"
                       type: "string"
+          Platforms:
+            description: "An array of supported platforms."
+            type: "array"
+            items:
+              type: "object"
+              properties:
+                Architecture:
```

Comment:
@nishanttotla Hopefully I'll get this all straight, but as far as I'm understanding, ARM has a specific need for architecture variants -- but they haven't completely ironed out what they'll call them. I've seen things like `arm64_ilp32_variant8` and `arm32_variant6`, as examples. The real point is that the arch isn't always the whole story when choosing the image to pull. Right now the engine only looks at `$GOARCH`  + `$GOOS` to decide which image to pull in the case of multi-arch, but I believe that in the future it will need to also consider the variant. 

@stevvooe can correct anything I've gotten wrong here.